controllers/routes.js

```diff
@@ -42,7 +42,8 @@ const routes = {
                     code: 0,
                     static: false,
                     live: true,
-                    isPostable: true
+                    isPostable: true,
+                    aliases: ['/home']
```

Comment:
Is this to mop up any traffic that might have linked to this?

Reply:
Yeah, you see this path in analytics (so presumably in google etc) so making it properly redirect would be nice.